content/references/rippled-api/public-rippled-methods/transaction-methods/submit.md

```diff
@@ -256,15 +256,23 @@ The response follows the [standard format][], with a successful result containin
-
-**Caution:** If this command results in an error messages, the message can contain the secret key from the request. (This is not a problem if the request contained a signed tx_blob instead.) Make sure that these errors are not visible to others.
+| `accepted`              | Boolean | _(Omitted in sign-and-submit mode)_ The value `true` indicates that the transaction was applied, queued, broadcast, or kept for later. A value of `false` indicates that none of those happened, so the transaction cannot possibly succeed as long as you do not submit it again and have not already submitted it another time. [New in: rippled 1.5.0][] |
+| `account_sequence_available` | Number | _(Omitted in sign-and-submit mode)_ The next [Sequence Number][] available for this account after all pending and [queued](transaction-queue.html) transactions. [New in: rippled 1.5.0][] |
+| `account_sequence_next` | number  | _(Omitted in sign-and-submit mode)_ The next [Sequence Number][] for this account after all pending transactions, not counting transactions in the queue. [New in: rippled 1.5.0][] |
+| `applied`               | Boolean | _(Omitted in sign-and-submit mode)_ The value `true` indicates that this transaction was applied to the open ledger. In this case, the transaction is likely, but not guaranteed, to be validated in the next ledger version. [New in: rippled 1.5.0][] |
+| `broadcast`             | Boolean | _(Omitted in sign-and-submit mode)_ The value `true` indicates this transaction was broadcast to peer servers in the peer-to-peer XRP Ledger network. (Note: if the server has no peers, such as in [stand-alone mode](rippled-server-modes.html#reasons-to-run-a-rippled-server-in-stand-alone-mode), the server uses the value `true` for cases where it _would_ have broadcast the transaction.) The value `false` indicates the transaction was not broadcast to any other servers. [New in: rippled 1.5.0][] |
+| `kept`                  | Boolean | _(Omitted in sign-and-submit mode)_ The value `true` indicates that the transaction was kept to be retried later. [New in: rippled 1.5.0][] |
+| `queued`                | Boolean | _(Omitted in sign-and-submit mode)_ The value `true` indicates the transaction was put in the [Transaction Queue](transaction-queue.html), which means it is likely to be included in a future ledger version. [New in: rippled 1.5.0][] |
```

Comment:
What is the difference between queued and kept to be retried later? Might be useful to state the difference here.

Reply:
I believe the difference is that a `kept` transaction provisionally failed (and _not_ with a tec code) but it was cached to be retried over the next few ledgers. By comparison, a queued transaction was judged "likely to succeed" but wasn't applied yet because the open ledger was already full (and the transaction didn't meet the open ledger cost).

But I don't totally understand how the cache for failed transactions works (maybe @ximinez could explain it better)? In any case, I'd rather not complicate things by going into that level of detail in this parameter table.